cmd/tealdbg/localLedger.go

```diff
@@ -53,6 +69,25 @@ func makeAppLedger(
 	apps := []basics.AppIndex{appIdx}
 	apps = append(apps, txn.Txn.ForeignApps...)
 
+	// first populate balances from the indexer
```

Comment:
Does this break existing behavior if `indexerURL` is not provided?

Reply:
should not be executed at all, added support for that